content/docs/reference/tutorials/kubernetes/tutorial-p8s-rollout.md

```diff
@@ -55,16 +55,16 @@ can see `canary-example-app` and `canary-staging-app` created.
 
 1. Now, install dependencies:
 
-   ```sh
-   npm install
-   ```
+    ```sh
+    $ npm install
```

Comment:
A few of the blocks on this page don't render the copy buttons, apparently because they're parented by an `li` and not a `div`. Changing the query to `:not(.code-hero) > div.highlight` seems to work, though, and _should_ be safe, I think.